### PR TITLE
Added fuzzy legcuffs

### PR DESCRIPTION
**Pull request by Heroman3003**
As a much nicer counterpart to regular ones. They replace regular in some instances of random item generation, closet in sex dungeon and restraints crate.

### CODE REVIEW

**Comment by Arokha** [comment]
Pics pls

**Comment by Heroman3003** [comment]
http://prntscr.com/j4ftok

File: code/game/objects/random/random.dm [hunk 1]
```diff
@@ -274,7 +274,7 @@
 				prob(1);/obj/item/clothing/suit/storage/vest/heavy/merc,
 				prob(1);/obj/item/weapon/beartrap,
 				prob(1);/obj/item/weapon/handcuffs/fuzzy,
-				prob(1);/obj/item/weapon/handcuffs/legcuffs,
+				prob(1);/obj/item/weapon/handcuffs/legcuffs/fuzzy,
```
Comment (Arokha) [inline code/game/objects/random/random.dm]: //VOREStation Edit

Comment (Heroman3003) [reply code/game/objects/random/random.dm]: There is an already existing edit comment above this whole list indicating that it is changed.

Comment (Arokha) [reply code/game/objects/random/random.dm]: Oof.

**Pull request merged**